fedora/Dockerfile.fedora30

```diff
@@ -0,0 +1,23 @@
+FROM fedora:30
```

Comment:
The build of fedora30 brokes with dependency hell in dnf:

``` bash
Error:
 Problem: problem with installed package fedora-release-container-30-1.noarch
  - package fedora-release-container-30-1.noarch conflicts with system-release provided by fedora-release-30-1.noarch
  - package fedora-release-30-1.noarch conflicts with system-release provided by fedora-release-container-30-1.noarch
  - package fedora-release-container-30-3.noarch conflicts with system-release provided by fedora-release-30-1.noarch
  - package fedora-release-30-1.noarch conflicts with system-release provided by fedora-release-container-30-3.noarch
  - conflicting requests
  - package fedora-release-container-30-1.noarch conflicts with system-release provided by fedora-release-30-3.noarch
  - package fedora-release-30-3.noarch conflicts with system-release provided by fedora-release-container-30-1.noarch
  - package fedora-release-30-3.noarch conflicts with system-release provided by fedora-release-container-30-3.noarch
  - package fedora-release-container-30-3.noarch conflicts with system-release provided by fedora-release-30-3.noarch
```

Reply:
This is a known issue: 
https://fedoraproject.org/wiki/Common_F30_bugs#Conflicts_between_fedora-release_packages_when_installing_package_groups

I've fixed it by temporary passing `--allowerasing` to dnf.

Reply:
Ok, ```--allowerasing``` is not a dangerous workaround, but it is usually not the preferred solution because it can cause later dependency problem or anything else that depends on this package can fail too.

I think we can go with this one after we test the container for its usage and evaluate if these packages being removed by ```--allowerasing``` won't be missed.